backend-tests/tests/test_create_artifact.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2018 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        https://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import requests
+import time
+
+from json import dumps
+
+from testutils.api.client import ApiClient
+from testutils.api import deployments, useradm
+from testutils.common import mongo, clean_mongo, create_org
+
+
+def test_create_artifact(mongo, clean_mongo):
+    tenant, username, password = "test.mender.io", "user@test.mender.io", "secret"
+    tenant = create_org(tenant, username, password)
+    r = ApiClient(useradm.URL_MGMT).call(
+        "POST", useradm.URL_LOGIN, auth=(username, password)
+    )
+    assert r.status_code == 200
+    assert r.text is not None
+    assert r.text != ""
+
+    filename = "/tests/run.sh"
```

Comment:
Should we write some data into the file? It would still be valid behavior if mender-artifact refused to create an update module with an empty script.

Reply:
@lluiscampos 

I'm using `/tests/run.sh` in the container as file to install; I just need a random file to upload; if you believe it is a better solution, I can write a temporary file and use that instead, it doesn't change anything from test's point of view.

Reply:
I misunderstood then! Now I see: that file actually exists and it is the tests wrapper :)

So for another reason now: but I still think we should change it. The test should be framework agnostic. What if I run the test locally (not from a container with a volume mounted in a very specific way)? Of what if we rename the script or move it somewhere else?

I think a temp file with just dummy content will do:
```
#!/bin/bash
true
```

Reply:
Ok, done!